### ISSUE TITLE
Add rule `enforce-interpolation-with-parenthesis`

### ISSUE THREAD

**Issue by vhoyer** [intlify/eslint-plugin-vue-i18n#126]
**Please describe what the rule should do:**

This rule would force people to use the `@:(interpolation.path)` syntax with the parenthesis.

This rule should prevent errors where we need to add a period (dot in the end of a sentence) right after an interpolation.

**What category should the rule belong to?**

- [ ] Enforces code style (layout)
- [x] Warns about a potential error (problem)
- [ ] Suggests an alternate way of doing something (suggestion)
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```js
{
  "foo": "bar",
  "baz": "I want @:foo" // <- this should get an error
}
```

```js
{
  "foo": "bar",
  "baz": "I want @:foo. My popcorn is gonna melt :thinking:" // <- this should get an error
}
```

**Additional context**
